server/controllers/class/addNewFeedback.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response, NextFunction } from 'express';
+import { addNewFeedbackQuery } from '../../queries';
+import { CustomError } from '../../utils';
+
+const addNewFeedback = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    if (!req.body.studentId) {
+      throw new CustomError(400, 'The data needed is not given!');
+    }
+
+    if (!req.body.feedback) {
+      throw new CustomError(400, 'The data needed is not given!');
+    }
```

Comment:
Validate the body, and for the student id, you don't get it from the body.
We did all the auth middleware so we can check who did the action, so you can get the id from it.